plugin/trino-kafka/src/main/java/io/trino/plugin/kafka/KafkaMetadata.java

```diff
@@ -100,6 +100,7 @@ public KafkaTableHandle getTableHandle(ConnectorSession session, SchemaTableName
                         kafkaTopicDescription.getMessage().flatMap(KafkaTopicFieldGroup::getDataSchema),
```

Comment:
Should we add docs changes for this commit?

Reply:
Yep! Added, lmk if you have any suggestions.